tests/system/providers/amazon/aws/example_emr_serverless.py

```diff
@@ -40,7 +40,6 @@
 
 with DAG(
     dag_id=DAG_ID,
-    schedule_interval='@once',
```

Comment:
By removing this doesn't it change the default to `@daily`?

Reply:
Hmm this one is actually inherited from @dstandish’s previous PR (there are several more changes like this there). I’m assuming yes it changes the schedule but it doesn’t really matter since this is only an example anyway.